server/test/handlers/account.test.js

```diff
@@ -94,7 +94,7 @@ describe('Test to check valid account creation', () => {
             const user = await accountHandler.createUserAccount(firstName, lastName, type, studentId, email, password, phone, userType, workerId, specialization, schoolId) ;
             
             // Assert
-            expect(user).toEqual(expect.objectContaining(studentUser));
```

Comment:
Hi Nancy,
This adjusts the test code so that `user` equals `user` which would always be true. I was wondering why these values were adjusted from `studentUser` and `workerUser` (line 132)?

Reply:
Hi Amy, I believe this was a part of resolving merging conflicts/updating with master. Else, I have made no edits to server/test/handlers/account.test.js. Would you suggest I add line 97 back into the file?

Reply:
Hi Amy,
Good catch!
My Action: I have reverted the incorrect commit and replaced line 97 with `expect(user).toEqual(expect.objectContaining(studentUser));`

You can see this change in line 97 of server/test/handlers/account.test.js

Please confirm if this takes care of your suggestion!